apps/api/src/schema/study-rooms.ts

```diff
@@ -1,30 +1,107 @@
 import { z } from "@hono/zod-openapi";
+import { timeRangeSchema } from "./lib";
 
 export const slotSchema = z.object({
   studyRoomId: z.string(),
-  start: z.string().datetime({ offset: true }).openapi({ example: "2021-01-06T08:00:00-08:00" }),
-  end: z.string().datetime({ offset: true }).openapi({ example: "2021-01-06T08:30:00-08:00" }),
-  isAvailable: z.boolean(),
+  start: z.string().datetime({ offset: true }).openapi({
+    description: "The start time of this slot",
+    example: "2021-01-06T08:00:00-08:00",
+  }),
+  end: z.string().datetime({ offset: true }).openapi({
+    description: "The end time of this slot",
+    example: "2021-01-06T08:30:00-08:00",
+  }),
+  url: z.string().openapi({
+    description:
+      "The link to this specific slot on the UCI Libraries site. " +
+      "This link focuses on the specific room containing this slot, eagerly selects this slot for booking, and jumps to the submit button.",
+    example: "https://spaces.lib.uci.edu/space/44704?date=2025-05-23T09:00Z#submit_times",
```

Comment:
woah that's cool

does this still exist if `!isAvailable`? probably insignificant either way since any reasonable consumer would not try to book it if it was not available

Reply:
the booking form never appears:
![image](https://github.com/user-attachments/assets/ff734e7f-2eed-4542-a7a6-483ad6bc0540)